### DISCUSSION TITLE
(Projects beta) Saving of changes should be default

### DISCUSSION THREAD

**Discussion by dtugend** [community/community#9043] [Projects and Issues]
I don't know if this is possible, but personally would prefer if **changes would be saved by default after editing**,

**combined with an (client-side) undo**.

I know this is a bit more risky, but it works pretty well on Asana (though they have server-side undo, but I guess that would be a jungle to implement, so that's why suggesting client-side undo).

Not sure if others prefer the same way and if it's feasible.

**Comment by chriskuech** [comment]
I would be grateful for any improvement on the current situation.  The current Save button's location is not obvious and I have to search for it every time.  IMO, overriding the `Cmd`/`Ctrl` + ` S` would be another acceptable solution to this problem.
